documentation/modules/new-migrating-virtual-machines-cli.adoc

```diff
@@ -795,13 +802,49 @@ EOF
 <5> Specify the name of the `NetworkMap` CR.
 <6> Specify a storage mapping even if the VMs to be migrated are not assigned with disk images. The mapping can be empty in this case.
 <7> Specify the name of the `StorageMap` CR.
-<8> By default, virtual network interface controllers (vNICs) change during the migration process. As a result, vNICs that are configured with a static IP linked to the interface name in the guest VM lose their IP.
+<8> By default, virtual network interface controllers (vNICs) change during the migration process. As a result, vNICs that are configured with a static IP address linked to the interface name in the guest VM lose their IP address. +
 To avoid this, set `preserveStaticIPs` to `true`. {project-short} issues a warning message about any VMs for which vNIC properties are missing. To retrieve any missing vNIC properties, run those VMs in vSphere in order for the vNIC properties to be reported to {project-short}.
-<9> You can use either the `id` or the `name` parameter to specify the source VMs.
-<10> Specify the VMware vSphere VM moRef. To retrieve the moRef, see xref:retrieving-vmware-moref_vmware[Retrieving a VMware vSphere moRef].
-<11> Optional: You can specify up to two hooks for a VM. Each hook must run during a separate migration step.
-<12> Specify the name of the `Hook` CR.
-<13> Allowed values are `PreHook`, before the migration plan starts, or `PostHook`, after the migration is complete.
+<9> Generates a network interface name for a target VM that overrides the template set in the plan.
+The template follows the Go template syntax and has access to the following variables:
+* `.NetworkName:` If the target network is `multus`, add the name of the Multus Network Attachment Definition. Otherwise, leave this variable empty.
+* `.NetworkNamespace`: If the target network is `multus`, add the namespace where the Multus Network Attachment Definition is located.
+* `.NetworkType`: Specifies the network type. Options: `multus` or `pod`.
+* `.NetworkIndex`: Sequential index of the network interface (0-based).
+Examples
+* `"net-{{.NetworkIndex}}"`
+* `{{if eq .NetworkType "pod"}}pod{{else}}multus-{{.NetworkIndex}}{{end}}"`
```

Comment:
```suggestion
* `{{if eq .NetworkType "pod"}}pod{{else}}multus-{{.NetworkIndex}}{{end}}"
```

Reply:
p.s.
didn't mark all the places with the unneeded '"' issue, there are more places that need this fix

Reply:
@yaacov The ` `s are used to format the text, so they need to stay according our documentation practices. But let me know if the " " are not supposed to entered by users.